cmd/pbstatuser/main.go

```diff
@@ -112,10 +137,13 @@ func checkSourceAvailabilityAzure(ctx context.Context) {
 		// TODO: check if source is avavliable - WIP
 		sr.Status = kafka.StatusAvaliable
 		chSend <- sr
+		totalAvailabilityCheckReqs.WithLabelValues("azure", string(sr.Status), "").Inc()
+		availabilityCheckReqsDuration.WithLabelValues("azure", "").Observe(float64(time.Since(start).Nanoseconds()) / 1000000)
```

Comment:
I am wondering why you don't do:

```suggestion
		availabilityCheckReqsDuration.WithLabelValues("azure", "").Observe(float64(time.Since(start).Milliseconds()))
```

Reply:
Oh I think I know why, you want float value not rounded integer.